wpilibNewCommands/src/test/native/cpp/frc2/command/MecanumControllerCommandTest.cpp

```diff
@@ -109,8 +110,7 @@ TEST_F(MecanumControllerCommandTest, ReachesReference) {
         m_rearLeftSpeed = rearLeft;
         m_frontRightSpeed = frontRight;
         m_rearRightSpeed = rearRight;
-      },
-      {&subsystem});
```

Comment:
Why was this removed? Also, I believe this was the only use of `subsystem`, so removing it makes the variable unused.

Reply:
Oops. Technically the subsystem variable isn't needed there so I removed it but meant to put it back after i replaced `Subsystem` with `TestSubsystem`